spec/fixtures/data_errors_integration/errors.yml

```diff
@@ -0,0 +1,84 @@
+rubocop:
```

Comment:
Maybe would be better to split this file into more smaller (by runner), what do you think?

Reply:
Also, a little bit strange name of the folder